include/boost/geometry/strategies/buffer/services.hpp

```diff
@@ -0,0 +1,46 @@
+// Boost.Geometry
+
+// Copyright (c) 2021, Oracle and/or its affiliates.
```

Comment:
Apparently the copyright message (with my name) is gone, can you keep the copyright of the moved messages?
I would like to keep seeing my name in the files I wrote :-)

Or actually, these files are probably brand new files, new syntactic sugar. Then it's fine, you can ignore this comment.

Reply:
Yes, they are new. You can always add something in this file together with copyright message. :)